packages/amplify-app/src/xcodeHelpers.js

```diff
@@ -5,7 +5,7 @@ const fs = require('fs-extra');
 async function getXcodeProjectDir() {
   const EXTENSION = '.xcodeproj';
   const files = fs.readdirSync(process.cwd());
-  const targetFiles = files.filter(function(file) {
+  const targetFiles = files.filter(function extenstionFilter(file) {
```

Comment:
use arrow functions for these things when possible, so this does not get rebound